src/gitManager.ts

```diff
@@ -128,7 +128,7 @@ export abstract class GitManager {
     */
     private simplify(tree: TreeItem[]): TreeItem[] {
         for (const node of tree) {
-            while (true) {
+            for (;;) {
```

Comment:
ESLint's no-constant-condition rule does not allow `while (true)`. Therefore, I have replaced it with `for (;;)` instead, per the following issue. If you don't like it, please disable no-constant-condition.
https://github.com/eslint/eslint/issues/5477

Reply:
Yeah I prefer `while(true)` If disabled the rule and reverted the change. Thanks for pointing out!